src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```
@@ -38,4 +38,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CentralPackageVersionsFileImported>true</CentralPackageVersionsFileImported>
   </PropertyGroup>
 
+  <!-- Extension point after Directory.Packages.props is imported. -->
+  <Import Project="$(CustomAfterDirectoryPackagesProps)" Condition="'$(CentralPackageVersionsFileImported)' == 'true' and '$(CustomAfterDirectoryPackagesProps)' != ''" />
```

This probably also needs an existence check.

```suggestion
  <Import Project="$(CustomAfterDirectoryPackagesProps)" Condition="'$(CentralPackageVersionsFileImported)' == 'true' and '$(CustomAfterDirectoryPackagesProps)' != '' and Exists('$(CustomAfterDirectoryPackagesProps)')" />
```

---

Do we need that? I.e. see https://github.com/dotnet/msbuild/blob/07a6d0d9cd3515987587b34dd8171c5c7372dcd9/src/Tasks/Microsoft.Common.props#L36

That doesn't have an Exists check either. If I set this property, I would want msbuild to tell me that the import fails when the file doesn't exist.